assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-selection-panel.scss

```diff
@@ -207,6 +207,50 @@
 				width: initial;
 			}
 		}
+
+		.googlesitekit-audience-creation-error-notice {
+			align-items: center;
+			background-color: $c-content-on-primary;
+			border: 2px solid $c-utility-error;
+			border-radius: $br-sm;
+			display: flex;
+			gap: 16px;
+			margin-bottom: 12px;
+			padding: 14px $grid-gap-phone;
+
+			svg {
+				color: $c-utility-error;
+				flex-shrink: 0;
+			}
+
+			p.googlesitekit-audience-creation-error-notice__title {
```

Comment:
According to Figma, we should have a gap of 4px between the title and the description. Was that intentionally not included?

Reply:
Updated.